### ISSUE TITLE
Set up reCAPTCHA 

### ISSUE THREAD

**Issue by wsalesky** [srophe/caesarea#74]
When there is a production site.

**Comment by davidamichelson** [comment]
@wsalesky Shoudl I do this now? What do I need to do?

**Comment by wsalesky** [comment]
Yes, you will need to register the site with reCAPTCHA, just as you did syriaca.org, then send me the key.

**Comment by davidamichelson** [comment]
Just sent the key. Please close out when ready.

**Comment by wsalesky** [comment]
Recaptcha is in place. Email function is not working.

**Comment by davidamichelson** [comment]
Sent credentials via email.

**Comment by davidamichelson** [comment]
@wsalesky Is this working? I tested today and it is not yet. Thanks!

**Comment by wsalesky** [comment]
@davidamichelson It is not working. The email function does not seem to work the way it used to, I am troubleshooting and will most likely have to email the list to find out what is going on. Hope to have it working by tomorrow!

**Comment by davidamichelson** [comment]
Thanks an no rush, we have a couple of weeks before the public release

**Comment by davidamichelson** [comment]
I will ask Joe Rife to do this.

**Comment by davidamichelson** [comment]
@wsalesky Sent an e-mail with gmail credentials.
